### ISSUE TITLE
Not working on Hyprland 0.40.0-1

### ISSUE THREAD

**Issue by iMithrellas** [sopa0/hyprsome#23]
Hi, I am unsure why, but this last update of Hyprland broke the functionality of your awesome tool. I am running the git version from the AUR. 

Hyprland tries execing hypsome workspace xx but nothing happens. I am usnure where to find Hyprsome logs so please if someone could direct me Id gladly provide them. 

For now I just downgraded to Hyprland 0.39.1-5 and it works great again.

**Comment by SpeckledFleebeedoo** [comment]
I think I am running into the same issue. Running the command in the terminal returns this:
```
❯ hyprsome workspace 2
thread 'main' panicked at 'A error occured while parsing the output from the hypr socket: IoError(Os { code: 2, kind: NotFound, message: "No such file or directory" })', /home/Speckled/.cargo/registry/src/index.crates.io-6f17d22bba15001f/hyprland-0.3.1/src/data/regular.rs:25:19
```

**Comment by sopa0** [comment]
Issue was fixed by #22 and new version is now available on cargo
